plugins/importexport/native/filter/NativeXmlSubmissionFilter.php

```diff
@@ -238,4 +243,22 @@ public function getImportFilter($elementName)
     {
         assert(false); // Subclasses should override
     }
+
+    /**
+     * @param string $stageName
+     * @param Submission|null $submission
+     * @return void
+     * @throws \Exception
+     */
+    public function handleSubmissionStage(string $stageName, ?Submission $submission): void
+    {
+        if ($stageName === "externalReview" or $stageName === "internalReview") {
+            $reviewRoundDao = DAORegistry::getDAO('ReviewRoundDAO');
+            $reviewRound = $reviewRoundDao->build(
+                $submission->getId(),
+                $submission->getData('stageId'),
+                ReviewRound::REVIEW_ROUND_STATUS_PENDING_REVIEWERS,
```

Comment:
Here the parameter reviewRound = 1 is missing

Reply:
@bozana  thanks! added!